### PR TITLE
Summary of ADADELTA

### PR DESCRIPTION
**Pull request by Shiviagarw**

### CODE REVIEW

File: _summaries/ShiviAgarwal_Adadelta.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+ADADELTA: AN ADAPTIVE LEARNING RATE METHOD, Zeiler1; 2012 - Summary
+
+author:	Shivi Agarwal
+
+score:	 / 10
```
Comment (mattkelleher) [inline _summaries/ShiviAgarwal_Adadelta.md:5]: Add your personal score for this paper and the ADADELTA learning introduced

File: _summaries/ShiviAgarwal_Adadelta.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+ADADELTA 
+
+
+•	What is the core idea?
+The paper introduces ADADELTA a learning rate used in gradient descendent algorithm for optimization of neural network.
+ADADELTA is computed on a per-dimension basis using only first order information.
+ADADELTA is better to other learning rates because it is robust to large gradients, noise and architecture choice and it requires minimal calculation over gradient descent.
```
Comment (mattkelleher) [inline _summaries/ShiviAgarwal_Adadelta.md:13]: What, if any, are the drawbacks? Based on this statement and the following statement about how it addresses the two main issues in ADAGRAD it seems like ADADELTA is really good. Are there any scenarios or tasks where you might expect it to struggle ?

File: _summaries/ShiviAgarwal_Adadelta.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ADADELTA performs better than ADAGRAD as it doesn’t decay the learning rate. In ADAGRAD due to continuous accumulation of squared denominator the learning rate eventually decreases to zero and stops training. ADAGRAD also requires to manually select the global learning rates which is not needed in ADADELTA.
+ADADELTA uses the window of past gradients that are accumulated to be some fixed size.
+
+•	How is it realized (technically)?
+The ADADELTA is defined by the function:
+Using accumulation over window it is defined as : 
+       ∆xt = − η RMS[g]t g 
+       
+Using Correct Units with Hessian Approximation:
+		∆xt = − RMS[∆x]t−1 /RMS[g]t gt
```
Comment (mattkelleher) [inline _summaries/ShiviAgarwal_Adadelta.md:23]: might be good to add a key help readers more easily understand what the different variables mean in these equations

File: _summaries/ShiviAgarwal_Adadelta.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+•	How well does the paper perform?
+The ADADELTA Learning Rate was  tested on multiple tasks, such as the MNIST classification (handwritten digits dataset), Speech Data and outperformed other methods in all tasks.
```
Comment (mattkelleher) [inline _summaries/ShiviAgarwal_Adadelta.md:26]: What methods did it outperform? Could you mention a few specifically?

File: _summaries/ShiviAgarwal_Adadelta.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    
+    
+
+
+<img src="ADADelta1.png">
+
+<img src="Adadelta2.png">
+
+
+TL;DR
```
Comment (mattkelleher) [inline _summaries/ShiviAgarwal_Adadelta.md:43]: might want to mention some key features in the TLDR. (ex. fixed window size etc etc) The current bullets are a bit vague

**Comment by zhaoyue-zephyrus** [comment]
@Shiviagarw  Could you please use the provided format -> https://github.com/395t/review-assignments/blob/main/_summaries/zeiler2012adadelta_2.md?

**Comment by zhaoyue-zephyrus** [comment]
@Shiviagarw  Can you edit the file https://github.com/395t/review-assignments/blob/main/_summaries/zeiler2012adadelta_2.md directly (replace the title, author, score item and put your reviews) instead of creating a new file?

**Comment by zhaoyue-zephyrus** [comment]
Since class is tomorrow, I am going to merge it first and rename manually. Please add your figures with proper naming via a new PR.

**Pull request merged**